ffmpy.py

```diff
@@ -60,7 +60,7 @@ def __init__(
     def __repr__(self):
         return "<{0!r} {1!r}>".format(self.__class__.__name__, self.cmd)
 
-    def run(self, input_data=None, stdout=None, stderr=None, env=None):
+    def run(self, input_data=None, stdout=None, stderr=None, env=None, creationflags=subprocess.CREATE_NEW_CONSOLE):
```

Comment:
Why does there need to be a default for `creationflags`? Could we leave it to the user to define the `creationflags` they need?

Reply:
I included it to reflect the default value assigned to _creationflags_ in _subprocess.run()_. From the [docs](https://docs.python.org/3/library/subprocess.html):

_class subprocess.Popen(args, bufsize=- 1, executable=None, stdin=None, stdout=None, stderr=None, preexec_fn=None, close_fds=True, shell=False, cwd=None, env=None, universal_newlines=None, startupinfo=None, creationflags=0, restore_signals=True, start_new_session=False, pass_fds=(), *, group=None, extra_groups=None, user=None, umask=- 1, encoding=None, errors=None, text=None, pipesize=- 1)_

**subprocess.CREATE_NEW_CONSOLE**
The new process has a new console, instead of inheriting its parent’s console (the default).

Reply:
I should add that the default was added so that no error would be thrown if a user just ran `ffmpy.run()` with no input parameters. 

Also, the default could instead be `creationflag=0` was is functionally the equivalent of `creationflag=subprocess.CREATE_NEW_CONSOLE`. I used the latter so that people could more easily and contextually look up the correct `creationflag` for the use case.